scripts/opentelemetry/appsignals_custom_otel_setup/custom-opentelemetry.yaml

```diff
@@ -0,0 +1,591 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector-role
+rules:
+  - apiGroups: [""]
+    resources: ["pods", "nodes", "namespaces", "endpoints"]
+    verbs: ["list", "watch", "get"]
+  - apiGroups: [""]
+    resources: ["services"]
+    verbs: ["list", "watch"]
+  - apiGroups: ["apps"]
+    resources: ["replicasets", "daemonsets", "deployments", "statefulsets"]
+    verbs: ["list", "watch", "get"]
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["list", "watch"]
+  - apiGroups: [""]
+    resources: ["nodes/proxy"]
+    verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["nodes/stats", "configmaps", "events"]
+    verbs: ["create", "get"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["update"]
+
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: otel-collector-role-binding
+subjects:
+  - kind: ServiceAccount
+    name: applicationsignals-otel-collector
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: otel-collector-role
+  apiGroup: rbac.authorization.k8s.io
+---
+
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: applicationsignals-otel
+spec:
+#  mode: daemonset
```

Comment:
Since we have defined ServiceAccount `applicationsignals-otel-collector`, we could also provide an example to add the service account name to the `OpenTelemetryCollector` like:
```
  ...
  image: public.ecr.aws/d8u3t5w4/appsignals-otel-collector:latest
  # serviceAccount: applicationsignals-otel-collector
  config: |
  ...
```

Reply:
I think OTel Operator will apply `serviceAccount: applicationsignals-otel-collector` for the collector automatically based on `metadata/name` given.  But I agree we can explicitly add it in yaml file so users can know what is the SA immediately.